### DISCUSSION TITLE
MHI2_ER_VWG11_P3301 patch

### DISCUSSION THREAD

**Discussion by Autjeg** [Mr-MIBonk/M.I.B._More-Incredible-Bash#144] [Q&A]
hello, 
I have tried to patch my MIB pro with More incredible bash to activate the performance menu.
I have downloaded the all patch zip file but unfortunatly don't have my MIB reference inside !

as you said in the menu, 'asck for path if needed', here my mib backup

https://mega.nz/file/mgBAgKyb#jpGH1KrLuDqo2FcTpxCS_RL4t3aLHWnlIWpedgHvqxU

thenks for all the wonderfull stuff you do !

**Comment by Autjeg** [comment]
Hi again is it still possible to have a missing patch?

**Reply by harman-f** [reply]
Yes, that FW does not have a patch
People normally just update and patch

**Reply by Autjeg** [reply]
Hi harman.
Sorry I don't really understand.
You advise me to upgrade to a version
Then patch..

Is that true?

**Reply by harman-f** [reply]
yes, update to latest G11 and patch that.

mibsolution.one

**Reply by Autjeg** [reply]
OK thanks i will try to find the last update.. 😁

**Reply by Autjeg** [reply]
Sorry again. Is there other link to download.. It gonna takes me 6 days from mib solution.one
Is that normal? 🤔

**Reply by harman-f** [reply]
use download manager --> free download manager

**Reply by Autjeg** [reply]
hello, I update my Mib to MHI2_ER_VWG11_K3342_MU1427, all works fine !
but when I want to patch, I have error sayin doesn't find the file.

I had copy the content of the patch folder MHI2_ER_VWG11_K3342_MU1427_PATCH into the "patches" folder of the sd card..

here the log file, the starnge thing is it search a .ifs file with a different name...

thanks for your help

[MU1427-JTH-07614.03.173405064D.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/10995823/MU1427-JTH-07614.03.173405064D.log)

**Reply by harman-f** [reply]
looks like you did not copy the patch files to SD

**Reply by Autjeg** [reply]
Yes.. Looks like. But ![20230317_083749.jpg](https://user-images.githubusercontent.com/127685990/225842118-04dfde64-7d61-43be-a781-c116b06348f7.jpg)

**Reply by Autjeg** [reply]
Ho maybe.. I must keep files in the directory patch name.. And not directly on the patches root folder

**Reply by harman-f** [reply]
https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/wiki/Prepare-M.I.B-SD-card-&-install-M.I.B#add-patch-files

Just follow the manual!

**Reply by Autjeg** [reply]
Thanks. So much.. Sorry I'm not very used with the navigation on github.com..

Will do this as soon as the car is back home 😁
